### ISSUE TITLE
Looping city lookup?

### ISSUE THREAD

**Issue by snuggs** [sneakyhead/sneakyhead#28]
@RobertChristopher @mrbernnz  know where this is coming from on a full test run? Keeps looping cities. Think it's coming from the proxy service. ![capture d ecran 2015-12-26 a 13 43 48](https://cloud.githubusercontent.com/assets/38223/12007525/ac0436b4-abd6-11e5-9fd4-d276cdde773a.png)

**Comment by RobertChristopher** [comment]
Yes that is just some code looping proxies. Included that months ago. Originally it was the latency code you and tom had written but I refactored it.

**Comment by snuggs** [comment]
@RobertChristopher We should disable it (at least in tests. Looks like it works LOL)

**Comment by RobertChristopher** [comment]
Yep it does not even serve a purpose and does not have any valid tests really supporting it. It's just there to be there (:
